frontend/src/components/SummaryDeclaration.vue

```diff
@@ -99,10 +99,9 @@
                 </v-expansion-panel>
               </div>
 
-              <div v-for="(facility, index) in summaryModel?.facilities" :key="facility?.facilityId" class="special">
+              <div v-for="facility in facilities" :key="facility?.facilityId" class="special">
                 <v-skeleton-loader
-                  v-if="isSummaryLoading[index]"
-                  :loading="isSummaryLoading[index]"
+                  v-if="!isSummaryComplete"
```

Comment:
This is not the correct flag to use. Currently, if the summary isn't complete then we get a v-skeleton-loader for each facility that never goes away.

Reply:
We will be looking at removing this flag entirely.

Reply:
Loader removed